ostelco-ios-client/Storyboards/Home.storyboard

```diff
@@ -19,7 +19,7 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="NWm-c7-dac">
+                            <scrollView verifyAmbiguity="off" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="NWm-c7-dac">
```

Comment:
why turn off verifying ambiguity here?

Reply:
I am not sure how to fix the red mark in IB, otherwise. During runtime, I don't see anything wrong with the UI

Reply:
OK, I'll take a poke at this once it's been merged. Constraints are kind of weird with scroll views - it has to know both the size of its frame plus the size of the contents within it. I'll see if I can dig up a good tutorial on it in the morning, it took me a stupidly long time to learn how it worked so this is definitely fine for now.

Reply:
thanks

Reply:
This is at least a good place to start (and this video is actually free) https://www.raywenderlich.com/3911-scroll-view-school/lessons/6